### PR TITLE
Affected User Flow Navigation Bar Updates

### PR DESCRIPTION
**Pull request by mattThousand**
#### Why:
We'd like our navigation bar conventions to be consistent across the app

#### This commit:
This commit swaps out the preexisting navbar (custom back button, `x` icon), with the navbar used elsewhere in the app (system back button), and adds a `Done` button to the "Guidance" screen so that the user can return to the exposure history screen once they have completed the affected user flow.

Oct-13-2020 21-51-21

### CODE REVIEW

File: src/SelfAssessment/Guidance.tsx [hunk 1]
```diff
@@ -183,8 +183,6 @@ const Guidance: FunctionComponent<GuidanceProps> = ({
     }
   }
 
-  const displayFindATestCenter = Boolean(findATestCenterUrl)
```
Comment (johnschoeman) [inline src/SelfAssessment/Guidance.tsx]: Why was this removed? what happens if the jurisdiction does not have a test center url

**Review by johnschoeman — COMMENTED** [review]
Code looks good, but looks like we are no longer respecting the safe area view?

File: src/SelfAssessment/Guidance.tsx (hunk 1 above)
Comment (mattThousand) [reply src/SelfAssessment/Guidance.tsx]: good catch 👍

**Review by johnschoeman — APPROVED** [review]
💯 

The gif still shows content going behind the notch, did we confirm if this is still an issue here?
